### DISCUSSION TITLE
Values are being parsed wrong when publishing msg PoseWithCovarianceStamped

### DISCUSSION THREAD

**Discussion by corcasta** [foxglove/community#731] [Q&A]
When using the internal publishing tool that foxglove studio provides, shifts all the values from one key to the next key in json format. 

![Screenshot from 2023-06-04 16-32-24](https://github.com/foxglove/community/assets/53949604/8fa5d1ac-f47f-415e-ab7b-cad0b28382c6)

**Comment by jtbandes** [comment]
Moved to https://github.com/foxglove/studio/issues/6190
